cfgov/jinja2/v1/_includes/organisms/job-listing-list.html

```diff
@@ -25,22 +25,24 @@
     <h3>
         {{ heading or 'Career Openings' }}
     </h3>
-    <ul class="m-list m-list__unstyled">
+    <ul class="m-list m-list__unstyled m-list__links">
         {% for job in careers %}
         <li class="m-list_item">
             <a class="m-list_link"
-               href="{{ get_protected_url(job) }}">{{ job.title }}</a>
-            <p class="a-date">
-                CLOSING
-                {% import 'macros/time.html' as time %}
-                {{ time.render(job.close_date, {'date': true}) }}
-            </p>
+               href="{{ get_protected_url(job) }}">{{ job.title }}
+               <span class="m-list_link-subtext">
+                   CLOSING
```

Comment:
It would be better for accessibility if we used `text-transform: uppercase;` to set this text to display in all-caps. Screen readers will sometimes read all-caps things as acronyms, spelling out each letter.

Reply:
The text will switch to sentence-case in [this PR](https://github.com/cfpb/cfgov-refresh/pull/3680), which changes the date formatting.

Reply:
Great!